README.md

```diff
@@ -0,0 +1,129 @@
+# StackStorm in Docker containers
+
+The initial container configuration is as follows:
+
+ - st2-upstart (st2 + st2web + sshd)
+ - mongo
+ - rabbitmq
+
+## Usage
+
+Build the st2-upstart image:
+
+  ```
+  make build
+  ```
+
+Start the docker environment:
+
+  ```
+  docker-compose up
+  ```
+
+Install the SSH keys locally, and setup ST2 user (this only needs to be run once per `docker-compose up`):
+
+*NOTE: The default values for `ST2USER` and `ST2PASSWORD` are `st2admin` and `Ch@ngeMe`
+respectively. You only need to specify these variables if you do not want to use the default
+values.*
+
+  ```
+  make setup ST2USER=st2admin ST2PASSWORD=Ch@ngeMe
+  ```
+
+Use either `ssh` or `docker exec` to connect to the st2-upstart container:
+
+  ```
+  ssh -i ~/.ssh/id_busybee root@localhost
+  ```
+
+  ```
+  docker exec -it st2docker_st2-upstart_1 /bin/bash
+  ```
+
+Once connected to the st2-upstart container, verify that `st2-self-check` passes successfully.
```

Comment:
This `st2-self-check` a super-agressive script, we should keep it for ourselves and not 
Please remove from README at some later time when it we are closer to "ready".